examples/components-macos/app/project.yml

```diff
@@ -18,8 +18,8 @@ targets:
       - Sources
     settings:
       HEADER_SEARCH_PATHS: '$(STDLIB_PATH)'
-      LIBRARY_SEARCH_PATHS: '$(STDLIB_PATH)'
-      OTHER_LDFLAGS: ' -framework Cocoa'
+      LIBRARY_SEARCH_PATHS: '$(STDLIB_PATH) /usr/local/lib/'
```

Comment:
What's depending on `gmp`?

Unless we want to tell users to install it separately, we'll need to depend on https://github.com/esy-packages/esy-gmp. I'll fix it to give us cflags/libs variables.

Reply:
@rauanmayemir `zarith`, a transitive dependency of `httpkit`.